.github/CODEOWNERS

```diff
@@ -1,3 +1 @@
-* @ocefpaf
-* @molinav
-* @ReimarBauer
+* @molinav @ocefpaf
```

Comment:
Here you disappear from the `CODEOWNERS` file, I guess it should remain as it was before.